botocore/client.py

```diff
@@ -522,7 +530,7 @@ def _service_model(self):
         return self.meta.service_model
 
     def _make_api_call(self, operation_name, api_params):
-        request_context = {}
+        request_context = {'client_region': self.meta.region_name}
```

Comment:
I would rather change that to be snake case than to make everything else camel case